### PR TITLE
MNTOR-1106: Add non-auth email preview

### PR DESCRIPTION
**Pull request by flozia**
# References: 
Jira: [MNTOR-1106](https://mozilla-hub.atlassian.net/browse/MNTOR-1106)

# Description

This adds a view-only version of the email previewer and aims to support people outside our team to easily access them.

# Screenshot

mntor-1106-screenshot

# How to test

Visit the Monitor website and navigate to `/preview/emails`.

# Checklist (Definition of Done)
- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [x] I've added a unit test to test for potential regressions of this bug.
- [x] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: src/views/main.js [hunk 1]
```diff
@@ -47,7 +48,11 @@ const mainLayout = data => `
     <!-- End Google Tag Manager -->
     </head>
   <body>
-    ${['landing', 'notFound'].includes(data.partial.name) ? landingHeader(data) : mainHeader(data)}
+    ${
+      showLandingHeaderFor.includes(data.partial.name)
+        ? landingHeader(data)
+        : mainHeader(data)
+    }
```
Comment (toufali) [inline src/views/main.js:55]: Nice!  I mentioned in one of @Vinnl 's PRs for error page that we should eventually create a new layout specific to non-auth pages (see MNTOR-1254), but this is a good improvement for the meantime, thanks!

**Review by toufali — COMMENTED** [review]
Lookin good @flozia.  I mostly tested functionality in-browser and glazed over the code.  Only thing I noticed: "Email verification", "Monthly unresolved", and "Signup report" all render with the same header text for me – is that intentional?

**Comment by flozia** [comment]
> Only thing I noticed: "Email verification", "Monthly unresolved", and "Signup report" all render with the same header text for me – is that intentional?

Thanks for catching this one! That’s not intentional and a display issue with the previewer — fixed by [b18f133](https://github.com/mozilla/blurts-server/pull/2850/commits/b18f133d334cfdc98f700b85a5000676ddb76d1c).

**Review by rhelmer — APPROVED** [review]

File: src/routes/index.js [hunk 2]
```diff
@@ -3,29 +3,35 @@
...
-router.use('/user', userRoutes)
+router.get('*/dialog/:name', dialog)
+
+router.use('/', dockerFlowRoutes)
 router.use('/admin', adminRoutes)
-router.use('/api/v1/user/', userApiRoutes)
 router.use('/api/v1/hibp/', hibpApiRoutes)
+router.use('/api/v1/user/', userApiRoutes)
+router.use('/oauth', authRoutes)
+router.use('/preview', previewRoutes)
```
[the remark this answers is not in the document]
Comment (flozia) [reply src/routes/index.js:32]: @mansaj do we build Heroku as `dev` or `prod`?

[the remark this answers is not in the document]
Comment (flozia) [reply src/routes/index.js:32]: It seems like we are building Heroku in prod. I created https://mozilla-hub.atlassian.net/browse/MNTOR-1264 so we can determine what the needs of the localization team are and then decide on what to do.

**Pull request merged**